CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.44.0 / 2020-12-02
+
+* [FEATURE] Add Slack, WeChat, Email, Pushover, and VictorOps receiver types to AlertmanagerConfig CRD. #3618 #3619 #3692 #3697 #3701
```

Comment:
Nit: I missed these just by reading these, wdyt if we list them individually instead? Any reason against it?

Reply:
Makes sense. Fixed